docs/the-new-architecture/landing-page.md

```diff
@@ -1,32 +1,71 @@
 
-To **create a new app** using the New Architecture, head over to [Creating a New Architecture App](use-app-template), which will get you up and running in a few quick steps with the new app template.
+In particular, synchronous rendering is a huge experience win for the New Architecture. Synchronous layout of your React views eliminates a class of bugs related to layout jumps and race conditions related to invalid layout data. Long-standing bugs with list positions and virtualization can be fixed holistically.
 
-To **migrate an existing app** to the New Architecture, follow [Adopting the New Architecture](../new-architecture-intro).
+### Support for React 18 concurrent features
 
-### For Library Maintainers
+The New Architecture brings support for [concurrent features in React](https://react.dev/blog/2022/03/29/react-v18#what-is-concurrent-react) like interruptible rendering to allow for a better user experience, especially when there are frequent UI updates.
 
-First, read up on the core concepts outlined in the [Pillars](pillars) section.
+The following compares rendering between the legacy and New Architecture in UI where the state of how many tiles to render is updated frequently by the slider value. You'll notice that the concurrent render will interrupt stale rendering work to acheive the finalized UI state faster.
 
-Then, for a **how-to guide** on supporting the New Architecture, check out the [Migration](../new-architecture-library-intro) guide.
+<table>
+<tr>
+ <td>
+<img src="https://github.com/facebook/react-native-website/assets/1309636/1cbd2d34-0639-4c20-922b-abb43e951c07" />
+</td>
+ <td>
+<img src="https://github.com/facebook/react-native-website/assets/1309636/a38cc835-0056-44e9-ab9d-4139752cd630" />
+</td>
+</tr>
 
-For information on **supporting both the Old and New Architectures**, see the [Backwards Compatibility](backward-compatibility) guide.
+</table>
+
+### Lower overhead for JS and native interoperability
+
+The New Architecture removes the use of a serializing message queue between JavaScript and native and replaces it with JavaScript Interface (JSI). JSI is an interface that allows JavaScript to hold a reference to a C++ object and vice-versa. With a memory reference, you can directly invoke methods without serialization cost.
+
+[VisionCamera](https://github.com/mrousavy/react-native-vision-camera), a React Native library, processes video frames in JavaScript in real-time. A typical video frame can be 10MB in size and can amount to 1GB per second depending on the frame rate. Passing that amount of data through the legacy messaging queue would make this experience unusable.
+
+Beyond high-volume data processing, JSI adoption in the New Architecture removes this class of serialization work from all native-JavaScript interop.
+
```

Comment:
```suggestion

### Compile-time code generation

CodeGen, an important part of the new architecture, generates types for the bindings between JavaScript and native TurboModules.

Using this generated type information, parameters and return values from native TurboModules receive proper nullability annotations as well as type declarations, resulting in potentially safer code.

Additionally, over-the-air updates can be automated easier, since CodeGen can be used to determine whether a native module's call-signature has been changed and a native update is required.

```

Something along the lines of this might also be good mentioning? CodeGen is pretty dope once it matures imo

Reply:
I agree codegen is pretty neat, do you feel like it's more interesting than the other points mentioned? I want to keep this section relatively short and we can always go into more depth on the working group.

Reply:
I don't think it is _more interesting_ than the other points mentioned, but I do think that this allows for many possibilities such as OTA, and in general a much safer native interop. This has been an issue a lot in many libraries, where e.g. the JS bundle is out of sync with native code.

But I agree that the other points are more interesting and this _could_ be left out, your call :)